src/openms/include/OpenMS/FORMAT/DATAACCESS/SiriusMzTabWriter.h

```diff
@@ -60,6 +60,7 @@ namespace OpenMS
       double isoscore;
       int explainedpeaks;
       double explainedintensity;
+      String mzml;
```

Comment:
add comments for fields - is mzml also a line of the sirius output? comment says so. And it doesn not return an mzTab!